swarm/network/simulations/overlay.go

```diff
@@ -67,23 +86,39 @@ func (s *Simulation) NewService(ctx *adapters.ServiceContext) (node.Service, err
 	return network.NewBzz(config, kad, store, nil, nil), nil
 }
 
-// var server
-func main() {
-	flag.Parse()
```

Comment:
This still needs to be called.

Reply:
Actually it's called from the `swarm/testutil/flags.go` file via the import....isn't that enough?

Reply:
Please read the comment below which tries to suggest a way to reach  agreement